### PR TITLE
mvp/feat/#2966 env bulk func tests

### PR DESCRIPTION
**Pull request by sri-miriyala**
Adding tests for negative scenarios and corresponding rules.

### CODE REVIEW

**Comment by sri-miriyala** [comment]
Screen Shot 2022-10-20 at 10 59 28 AM

File: collections/hub/sdk-bulk/bulk-happy-path.json [hunk 1]
```diff
@@ -53,51 +54,52 @@
...
                 },
                 "amountType": "SEND",
-                "currency": "USD",
-                "amount": "100"
+                "currency": "{$inputs.SIM3_CURRENCY}",
+                "amount": "678.91"
               }
             ]
           },
+          "url": "{$inputs.SIM1_SDK_BACKEND_ENDPOINT}",
```
Comment (vijayg10) [inline collections/hub/sdk-bulk/bulk-happy-path.json:102]: The "Override with Custom URL" should not be checked in all the test cases. i.e, you need to remove this parameter

**Review by vijayg10 — CHANGES_REQUESTED** [review]

File: collections/hub/sdk-bulk/bulk-happy-path.json (hunk 1 above)
Comment (vijayg10) [reply collections/hub/sdk-bulk/bulk-happy-path.json:102]: Please remove this in all the places

File: collections/hub/sdk-bulk/bulk-happy-path.json [hunk 2]
```diff
@@ -53,51 +54,52 @@
                 "to": {
                   "partyIdInfo": {
                     "partyIdType": "MSISDN",
-                    "partyIdentifier": "56781234001"
+                    "partyIdentifier": "{$inputs.SIM2_MSISDN_1}"
```
Comment (vijayg10) [inline collections/hub/sdk-bulk/bulk-happy-path.json:57]: Can we call this as TTKSIM1_MSISDN_1 instead? Because we are using SIM1, SIM2...etc already.
Please change SIM1 to TTKSIM1, SIM2 to TTKSIM2 in all the places

**Review by vijayg10 — CHANGES_REQUESTED** [review]

File: collections/hub/sdk-bulk/bulk-happy-path.json (hunk 2 above)
Comment (sri-miriyala) [reply collections/hub/sdk-bulk/bulk-happy-path.json:57]: I was hoping to leave it as generic name because different environments have different naming requirements for the sims. So if in a different env we had a requirement to name the sim as something different than TTKSIM, I felt leaving it as SIM is generic enough @vijayg10

File: collections/hub/sdk-bulk/bulk-happy-path.json (hunk 1 above)
Comment (sri-miriyala) [reply collections/hub/sdk-bulk/bulk-happy-path.json:102]: @vijayg10 ..if this is removed, what will be the default value of the URL?

Comment (vijayg10) [reply collections/hub/sdk-bulk/bulk-happy-path.json:102]: The default value is already configured in the TTK settings. So we don't need the URL in all the test cases.

File: collections/hub/sdk-bulk/bulk-happy-path.json (hunk 2 above)
Comment (vijayg10) [reply collections/hub/sdk-bulk/bulk-happy-path.json:57]: SIM1, SIM2 are already used for testfsp1, testfsp2...etc

File: collections/hub/sdk-bulk/bulk-happy-path.json (hunk 1 above)
Comment (sri-miriyala) [reply collections/hub/sdk-bulk/bulk-happy-path.json:102]: What is the default value? @vijayg10

Comment (sri-miriyala) [reply collections/hub/sdk-bulk/bulk-happy-path.json:102]: Also is there any harm in leaving this value? Because my plan is to use these collections both in local and in all the environments without any changes to the collections. In local env, we need this value. @vijayg10

**Review by kleyow — APPROVED** [review]

File: collections/hub/sdk-bulk/bulk-happy-path.json (hunk 2 above)
Comment (sri-miriyala) [reply collections/hub/sdk-bulk/bulk-happy-path.json]: @vijayg10...took care of this

**Review by vijayg10 — APPROVED** [review]
